plugins/de/opendiabetes/vault/plugin/importer/miband/MiBandNotifyImporter.java

```diff
@@ -168,9 +167,7 @@ protected void preprocessingIfNeeded(final String filePath) { }
                 this.notifyStatus(STATUS_IMPORTED_ENTRIES, "Successfully imported MiBand data to VaultEntries");
                 this.notifyStatus(STATUS_INTERPRETED_ENTRIES, "Interpreted MiBand data");
             } else {
-                LOG.log(Level.SEVERE, "Got no data from JSON import!");
```

Comment:
Warum hier das Log raus genommen?

Reply:
Je nachdem, die Exception wird ja so oder so in der Konsole ausgegeben, der Log dazu wäre redundant. Aber könnte sein, dass der Log noch irgendwo anders weitergegeben (gespeichert) wird, dann wäre es relevant. Ist das so?

Reply:
Uhm die Idee ist doch, dass die Anwendung aka GUI die Exception catched und dann handled.
Das Log ist davon ja unabhängig und sollte für ein Deployment ja auch von Console auf File geändert werden, würde ich sagen

Reply:
Ok dann nehm ich es wieder mit rein